Formula/vips.rb

```diff
@@ -56,7 +57,6 @@ def install
     args = %W[
       --disable-dependency-tracking
       --prefix=#{prefix}
-      --with-magick
```

Comment:
If I pass `--with-magick`, and `configure` can't find it, does it carry on as if I didn't ask to use magick, or does it error?

Reply:
It's equivalent to `test`, so there's no error. Though arguably there should be.